### PR TITLE
Fix for #3243

### PR DESCRIPTION
**Pull request by seriva**

### CODE REVIEW

**Review by sbbroot — DISMISSED** [review]

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh [hunk 1]
```diff
@@ -159,10 +160,36 @@ fi
...
+check_connection wget $(for file in $packagesfromurl; do echo "$file"; done)
+
+
+if [[ -z "${packagesfromurl}" ]]; then
+    echol "No packages from URL to download"
+else
+    # be verbose, show what will be downloaded
+    # TODO: this is the list of all files shows on every run, not only the files that will be downloaded this run
+    echol "Package from URL to be downloaded:"
+    cat -n <<< "${files}"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh]:
```suggestion
    cat -n <<< "${packagesfromurl}"
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh [hunk 2]
```diff
@@ -159,10 +160,36 @@ fi
...
+
+check_connection wget $(for file in $packagesfromurl; do echo "$file"; done)
+
+
+if [[ -z "${packagesfromurl}" ]]; then
+    echol "No packages from URL to download"
+else
+    # be verbose, show what will be downloaded
+    # TODO: this is the list of all files shows on every run, not only the files that will be downloaded this run
+    echol "Package from URL to be downloaded:"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh]:
```suggestion
    echol "Packages from URL to be downloaded:"
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh [hunk 3]
```diff
@@ -159,10 +160,36 @@ fi
...
+    cat -n <<< "${files}"
+
+    printf "\n"
+    # download files using wget
+    while IFS= read -r file; do
+        # download files, skip if exists
+        #wget --no-verbose --continue --directory-prefix="${dst_dir_files}" "${file}"
+        #wget --continue --show-progress --directory-prefix="${dst_dir_files}" "${file}"
+        download_file "${file}" "${dst_dir_packages}"
+    done <<< "${files}"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh]:
```suggestion
    done <<< "${packagesfromurl}"
```

**Comment by seriva** [comment]
@to-bar Good catches, copy+pasting at the end of the day is always a bad idear;)

**Review by to-bar — APPROVED** [review]

**Review by cicharka — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**